scene/3d/gi_probe.cpp

```diff
@@ -387,8 +387,10 @@ void GIProbe::_find_meshes(Node *p_at_node, List<PlotMesh> &plot_meshes) {
 	for (int i = 0; i < p_at_node->get_child_count(); i++) {
 
 		Node *child = p_at_node->get_child(i);
-		if (!child->get_owner())
-			continue; //maybe a helper
+		if (!Object::cast_to<Spatial>(child)) { // test for Spatial special case is required to allow for dynamically added Scenes at runtime
+			if (!child->get_owner())
+				continue; //maybe a helper
+		}
```

Comment:
I would just remove this whole check. 

`_find_meshses()` only cares about MeshInstances, which inherit from Spatial, so disabling the owner check for Spatials is the same as disabling it for any type of Node. I can't think of any case where un-owned Nodes would break baking, so I think it's safe to just remove the ownership test.